### DISCUSSION TITLE
+m,+n, is ,us,er and others as a combing charcters approach/font feature?

### DISCUSSION THREAD

**Discussion by afarlie** [psb1558/Junicode-font#120] [Ideas]
Prompted by:-
https://en.wikisource.org/wiki/Page:The_record_interpreter-_a_collection_of_abbreviations.djvu/13
The current template approach is not ideal.

Would it be possible to have a font-feature for typing a word in full, and having the font handle the contractions for rendering in the abbreviated forms Martin and Statutes of the Realm uses?

I.e  You type for example `<span style="font-features-settings: abbrvforms2>communis</span> but what is rendered is 
`com&x#304;n&304;&A76D;` without needing to setup  more complex combining marks?

**Comment by psb1558** [comment]
OpenType doesn't have features capable of doing that kind of complex transformation: it's designed to work on letters, not words. It's got features for substituting one letter for several and several letters for one, but not for substituting one string for another. While it might be possible to kludge up some kind of solution, it would be a very complex piece of programming for a job that would be trivial in Javascript or PHP.

**Comment by afarlie** [comment]
Thanks for a fast response indicating this isn't feasible.

**Reply by psb1558** [reply]
Sorry I can't deliver better news.
